shared/haml/course_explorer_table.haml

```diff
@@ -260,7 +260,7 @@
           .title
             Professional Learning
         .right{style: 'float:right; margin-top: -26px;'}
-          %a.linktag#professional-learning{:href => CDO.code_org_url('/educate/professional-learning')}
+          %a.linktag#professional-learning{:href => CDO.code_org_url('/educate/professional-learning'), :'aria-label' => "Learn more about Professional Learning Opportunities"}
```

Comment:
This view isn't translated

Reply:
This view renders differently for other languages, therefore translations are not used.